autoload/gitgutter/hunk.vim

```diff
@@ -300,22 +300,47 @@ function! s:preview(hunk_diff)
   let body_length = len(body)
   let previewheight = min([body_length, &previewheight])
 
-  silent! wincmd P
-  if !&previewwindow
-    noautocmd execute g:gitgutter_preview_win_location previewheight 'new'
-    set previewwindow
+  if has('nvim') && exists('*nvim_open_win')
+    let buf = nvim_create_buf(v:false, v:false)
+    let width = max(map(copy(lines), 'strdisplaywidth(v:val)'))
```

Comment:
You need `body` instead of `lines`.

Reply:
Right, I should use that. I was thinking of adding height offset in case width is greater than `&columns` so that the floating window would display wrapped lines.